build/commands/lib/util.js

```diff
@@ -62,7 +62,11 @@ async function applyPatches() {
   devtoolsFrontendPatchStatus.forEach(
     s => s.path = path.join('third_party', 'devtools-frontend', 'src', s.path))
   const allPatchStatus = [...chromiumPatchStatus, ...v8PatchStatus, ...catapultPatchStatus, ...devtoolsFrontendPatchStatus, ...ffmpegPatchStatus]
-  Log.allPatchStatus(allPatchStatus, 'Chromium')
+  if (printPatchFailuresInJson) {
+    Log.printLogFailuresToJson(allPatchStatus, config.braveCoreDir)
```

Comment:
you have `patchesPath` available here if you still wanted to strip out the `patches` part...

Reply:
I think it is fine to keep the `patch/` part of it.